There are still some warnings/errors to be sorted out:

```
2023-05-31T22:08:46.0320580Z qtmain.lib(qtmain_win.obj) : warning LNK4099: PDB '' was not found with 'qtmain.lib(qtmain_win.obj)' or at ''; linking object as if no debug info [D:\a\nano-node\nano-node\build\nano_wallet.vcxproj]
2023-05-31T22:08:46.0322226Z      Creating library D:/a/nano-node/nano-node/build/RelWithDebInfo/nano_wallet.lib and object D:/a/nano-node/nano-node/build/RelWithDebInfo/nano_wallet.exp
2023-05-31T22:08:57.5193797Z   nano_wallet.vcxproj -> D:\a\nano-node\nano-node\build\RelWithDebInfo\nano_wallet.exe
2023-05-31T22:08:57.6182497Z   Building Custom Rule D:/a/nano-node/nano-node/CMakeLists.txt
2023-05-31T22:08:57.8724624Z   -- Install configuration: "RelWithDebInfo"
2023-05-31T22:08:57.8725646Z   -- Installing: C:/Program Files/nano-node/./config-node.toml.sample
2023-05-31T22:08:57.8770223Z   -- Installing: C:/Program Files/nano-node/./config-rpc.toml.sample
2023-05-31T22:08:57.8786402Z   -- Installing: C:/Program Files/nano-node/./nano_wallet.exe
2023-05-31T22:08:57.9371211Z   -- Installing: C:/Program Files/nano-node/./nano_wallet.com
2023-05-31T22:08:58.0057344Z   -- Installing: C:/Program Files/nano-node/./vc_redist.x64.exe
2023-05-31T22:09:01.2483934Z   -- Installing: C:/Program Files/nano-node/./libGLESv2.dll
2023-05-31T22:09:01.5015803Z   -- Installing: C:/Program Files/nano-node/./Qt5Core.dll
2023-05-31T22:09:01.9540960Z   -- Installing: C:/Program Files/nano-node/./Qt5Gui.dll
2023-05-31T22:09:02.4595611Z   -- Installing: C:/Program Files/nano-node/./Qt5Widgets.dll
2023-05-31T22:09:02.8479047Z   -- Installing: C:/Program Files/nano-node/./Qt5WinExtras.dll
2023-05-31T22:09:02.8663403Z   -- Installing: C:/Program Files/nano-node/platforms/qwindows.dll
2023-05-31T22:09:02.9829183Z   -- Installing: C:/Program Files/nano-node/./api/flatbuffers
2023-05-31T22:09:02.9835863Z   -- Installing: C:/Program Files/nano-node/./api/flatbuffers/nanoapi.fbs
2023-05-31T22:09:02.9947460Z   -- Installing: C:/Program Files/nano-node/./nano_node.exe
2023-05-31T22:09:03.0883938Z   -- Installing: C:/Program Files/nano-node/./nano_rpc.exe
2023-05-31T22:09:03.3918053Z "Packaging NSIS"
2023-05-31T22:09:03.3939127Z '"\cpack.exe"' is not recognized as an internal or external command,
2023-05-31T22:09:03.3939917Z operable program or batch file.
2023-05-31T22:09:03.3940271Z "Packaging ZIP"
2023-05-31T22:09:03.3950447Z '"\cpack.exe"' is not recognized as an internal or external command,
2023-05-31T22:09:03.3950883Z operable program or batch file.
2023-05-31T22:09:04.8866646Z SignTool Error: Missing filename.
2023-05-31T22:09:04.9223608Z 
2023-05-31T22:09:05.2014733Z ##[error]Process completed with exit code 1.
2023-05-31T22:09:05.3199856Z Post job cleanup.
```

--- a/ci/actions/windows/configure.bat
+++ b/ci/actions/windows/configure.bat
@@ -10,7 +10,7 @@ cmake .. ^
   %CI% ^
   %ROCKS_LIB% ^
   -DPORTABLE=1 ^
-  -DQt5_DIR="c:\qt\5.13.1\msvc2017_64\lib\cmake\Qt5" ^
+  -DQt5_DIR="c:\qt\5.15.2\msvc2019_64\lib\cmake\Qt5" ^
   -DNANO_GUI=ON ^
   -DCMAKE_BUILD_TYPE=%BUILD_TYPE% ^
   -DACTIVE_NETWORK=nano_%NETWORK_CFG%_network ^

--- a/ci/actions/windows/install_deps.ps1
+++ b/ci/actions/windows/install_deps.ps1
@@ -49,8 +49,8 @@ function Get-RedirectedUri {
     }
 }
 $qt5_root = "c:\qt"
-$qt5base_url = Get-RedirectedUri "https://repo.nano.org/artifacts/5.13.1-0-201909031231qtbase-Windows-Windows_10-MSVC2017-Windows-Windows_10-X86_64.7z"
-$qt5winextra_url = Get-RedirectedUri "https://repo.nano.org/artifacts/5.13.1-0-201909031231qtwinextras-Windows-Windows_10-MSVC2017-Windows-Windows_10-X86_64.7z"
+$qt5base_url = Get-RedirectedUri "https://repo.nano.org/artifacts/5.15.2-0-202011130602qtbase-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64.7z"
+$qt5winextra_url = Get-RedirectedUri "https://repo.nano.org/artifacts/5.15.2-0-202011130602qtwinextras-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64.7z"
 $qt5base_artifact = "${env:TMP}\qt5base.7z"
 $qt5winextra_artifact = "${env:TMP}\qt5winextra.7z"
 

